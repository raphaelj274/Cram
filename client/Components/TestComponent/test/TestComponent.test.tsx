import React from 'react';
import { render } from '@testing-library/react-native';

import TestComponent from '../TestComponent'

it('TestComponent', () => {
  // arrange
  const component = render(<TestComponent/>).toJSON();

  // act
  // assert
  expect(component).toMatchSnapshot();
})

//TODO delete before we finish the project