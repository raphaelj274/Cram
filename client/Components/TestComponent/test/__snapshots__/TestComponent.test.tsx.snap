// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<View>
  <Text>
    Hello
  </Text>
</View>
`;

exports[`TestComponent 1`] = `
<View>
  <Text>
    Hello
  </Text>
</View>
`;
